command/agent/dns_test.go

```diff
@@ -3381,10 +3381,15 @@ func TestDNS_PreparedQuery_Failover(t *testing.T) {
 		if got, want := len(srv1.agent.WANMembers()), 2; got < want {
 			r.Fatalf("got %d WAN members want at least %d", got, want)
 		}
+		if got, want := len(srv2.agent.WANMembers()), 2; got < want {
```

Comment:
Should this be added here, too? https://github.com/hashicorp/consul/blob/8b4878e289ce4bf84eab257767c37ea4720b34e4/command/agent/dns_test.go#L1305-L1309